packages/flutter/lib/src/rendering/error.dart

```diff
@@ -9,8 +9,9 @@ import 'object.dart';
 
 const double _kMaxWidth = 100000.0;
 const double _kMaxHeight = 100000.0;
+
 // Line length to fit small phones without dynamically checking size.
-const String line = '\n────────────────────\n\n';
+const String _kLine = '\n────────────────────\n\n';
```

Comment:
while you're here we should probably also add another `\n` at the start there so it's more balanced.